test/python/opflow/test_aer_pauli_expectation.py

```diff
@@ -120,6 +121,18 @@ def test_pauli_expect_state_vector(self):
 
         np.testing.assert_array_almost_equal(sampled.eval(), [0, 0, 1, -1], decimal=1)
 
+    def test_pauli_expect_non_hermitian_state_vector(self):
+        """pauli expect state vector with non hermitian operator test"""
+        states_op = ListOp([One, Zero, Plus, Minus])
+
+        op = np.array([[0, 1], [2, 3]])
```

Comment:
Should we add a test for other primitives than `MatrixOp`? What about e.g. `1j * Y` as non-hermitian operator?

Reply:
A test case was added with the primitive PauliOp (I used 1j*X because with 1j*Y all the expected values for Plus, Minus, 0, 1 were null). The names of the two test cases were adapted consequently.